CameraOverlay/mqtt-test.py

```diff
@@ -0,0 +1,123 @@
+import sys
```

Comment:
Although very useful to have with this repository, I would remove this file since it is an exact copy of what we have in the data-acquisition-system repo. If I make a change to this test script in that repo, I would then need to change it here too!